### ISSUE TITLE
Open with advisory works should have some h1 before user clicks through the modal

### ISSUE THREAD

**Issue by davidpmccormick** [wellcomecollection/wellcomecollection.org#7545]
**What**
A work that has open with advisory content will display a modal on initial page load. This modal has to be dismissed before the main element of the page is displayed. The main element is what contains the `h1` for the work, so as long as the modal is present, there isn't an `h1` on the page. This is a potential accessibility problem.

We should consider what/where the appropriate `h1` should be for pages under these conditions.

**Where**
e.g. https://wellcomecollection.org/works/fa7pymra/items

**Comment by pollecuttn** [comment]
Also mentioned during the Hassell Inclusion accessibility audit August 2023 https://proficient-wallaby-dpwd.dovetailapp.com/insights/Content-advisory-missing-focus-14Rdq1E5MjtyyvQyvLFqZE

**Comment by rcantin-w** [comment]
Make the "Content advisory" the h1.
